sass/general.scss

```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+@import "varNmixins";
+@import "nav";
+@import "header";
+@import "main";
+@import "footer";
+@import "animations";
+
+//general styling:
+
+*{
+  margin: 0;
+  padding: 0;
+}
+
+html{
+  font-size: 62.5%;
+  overflow-x: hidden;
+}
+
+body{
+  @include flex($flex-col);
+  margin: inherit;
+  font-family: $font-family-main;
+  color: $white-clr;
+  min-width: 100vw;
+  max-width: 100vw;
+}
```

Comment:
Nie do końca rozumiem cel tego zabiegu: 
`  min-width: 100vw;
  max-width: 100vw;
`
Czemu nie wpisałaś po prostu:
` width: 100vw; `

Reply:
Ha, ja już teraz też nie, ale jak tak myślę, to wydaje mi się, że coś mi po prostu nie działało, tak jak miało i kombinowałam, bo czym zapomniałam usunąć, jak się okazało, że to niepotrzebny zabieg. Innego wytłumaczenia nie widzę, bo rzeczywiście bez sensu :grin: